src/arrayReverse.test.js

```diff
@@ -8,13 +8,16 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(Array.isArray(arrayReverse(['Mate', 'Academy']))).toBeTruthy();
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([''])).toEqual(['']);
```

Comment:
The test description mentions returning an empty string, but the test case checks for an array with an empty string. Ensure that the description and the test case align with the expected behavior.